### ISSUE TITLE
Changes needed in HTML and CSS @rexyagami

### ISSUE THREAD

**Issue by Astha0407** [incubateind/gdtc-Day2#74]

**Comment by rexyagami** [comment]
@Astha0407 can you explain the issue, please?

**Comment by Astha0407** [comment]
Aligned the heading in the center, padding in sections and labels in progress bar are incorrect @rexyagami

**Comment by rexyagami** [comment]
@Astha0407 label in the progress bar are correct, you just have to change the progress bar value and match the value with the label value. And yes the padding is incorrect you can find some hints in the CSS file

**Comment by Astha0407** [comment]
Yes @rexyagami  I have made the changes in CSS and changed the values of the progress bar matching to the label in HTML. and also made the Pull Request fixing all these issues

**Comment by rexyagami** [comment]
@Astha0407 Well Done :)
